test/unit/test_amqpclient.js

```diff
@@ -106,7 +106,7 @@ describe('AMQPClient', function() {
 
       return client.connect(mock_uri)
         .then(function () {
-          return client.createSender(queue, { name: 'queue_TX' });
+          return client.createSender(queue, { options: { name: 'queue_TX' } });
```

Comment:
I feel kind of like calling the key _inside_ the link policy `options` is a mistake, and it should be `attachParams` or something.